scalajvm/app/controllers/api/Quotes.scala

```diff
@@ -0,0 +1,22 @@
+package controllers.api
+
+import helpers.ActionWithTx
+import models.queries.QuoteQueries
+import play.api.mvc._
+import ru.org.codingteam.loglist.dto.QuoteDTO
+
+object Quotes extends Controller {
+  def getQuote(id: Long) = ActionWithTx { request =>
+    import request.dbSession
+
+    val probablyQuoteDTO = for {
+      quote <- QuoteQueries().getQuoteById(id)
+      quoteDTO <- Some(QuoteDTO(quote.id, quote.time.getMillis, quote.content, quote.rating))
```

Comment:
I don't very like this direct `Some` invocation. Wouldn't it be better to write something like `QuoteQueries().getQuoteById(id) map { quote => QuoteDTO(quote.id, quote.time.getMillis, quote.content, quote.rating) }`?